README.md

```diff
@@ -24,6 +10,18 @@ products:
 - vs-code
 ---
```

Comment:
I can't believe this HAS to go on the top, otherwise it doesn't render correctly